src/ontology/Makefile

```diff
@@ -1002,3 +1002,21 @@ bg-clean:
 
 succeed:
 	echo make succeeded
+	
+
+###### NICO MIGRATION: overwriting unsuccessful goals!
+
+../sparql/construct-ecs-from-xrefs.sparql:
+	echo "STRONG WARNING STRONG WARNING make $@ is skipped!" && touch $@
+	
+d2p.pro:
```

Comment:
harmless but uneccessary. You can simply remove all mention of this. It is not in the build graph for make all

Reply:
d2p.pro -> reports/semantic-xref-pairs.tsv -> $(ADDITIONAL_REPORT_FILES) -> $(REPORT_FILES) -> $(ARTEFACTS)